tk_barrydegraaff_zimbra_openpgp/tk_barrydegraaff_zimbra_openpgp.js

```diff
@@ -663,7 +762,7 @@ function(id, title, message) {
       "</td></tr><tr><td>" +
       tk_barrydegraaff_zimbra_openpgp.lang[tk_barrydegraaff_zimbra_openpgp.settings['language']][20]+":" +
       "</td><td>" +
-      "<input class=\"barrydegraaff_zimbra_openpgp-input\" id='passphraseInput' value='" + (this.getUserPropertyInfo("zimbra_openpgp_privatepass").value ? this.getUserPropertyInfo("zimbra_openpgp_privatepass").value : tk_barrydegraaff_zimbra_openpgp.prototype.pwgen()) + "'>" +
+      "<input class=\"barrydegraaff_zimbra_openpgp-input\" id='passphraseInput' value='" + tk_barrydegraaff_zimbra_openpgp.prototype.pwgen() + "'>" +
```

Comment:
in that case, I take it you mean 'people looking over the shoulders'
Please change the html input type to password, and add a html button to 'show password'

Date: Tue, 26 May 2015 09:25:44 -0700
From: notifications@github.com
To: pgp-zimlet@noreply.github.com
CC: info@barrydegraaff.tk
Subject: Re: [pgp-zimlet] Store encrypted passphrase locally (#95)

In tk_barrydegraaff_zimbra_openpgp/tk_barrydegraaff_zimbra_openpgp.js:

> @@ -663,7 +762,7 @@ function(id, title, message) {
>        "</td></tr><tr><td>" +
>        tk_barrydegraaff_zimbra_openpgp.lang[tk_barrydegraaff_zimbra_openpgp.settings['language']][20]+":" +
>        "</td><td>" +
> -      "<input class=\"barrydegraaff_zimbra_openpgp-input\" id='passphraseInput' value='" + (this.getUserPropertyInfo("zimbra_openpgp_privatepass").value ? this.getUserPropertyInfo("zimbra_openpgp_privatepass").value : tk_barrydegraaff_zimbra_openpgp.prototype.pwgen()) + "'>" +
> +      "<input class=\"barrydegraaff_zimbra_openpgp-input\" id='passphraseInput' value='" + tk_barrydegraaff_zimbra_openpgp.prototype.pwgen() + "'>" +

It is not secure when putting the passphrase in plain text into that input box.


—
Reply to this email directly or view it on GitHub.